shortcircuit-api/src/main/java/com/powsybl/shortcircuit/FaultParameters.java

```diff
@@ -24,15 +24,21 @@
  */
 public class FaultParameters {
 
+    // VERSION = 1.0 withLimitViolations, withVoltageMap, withFeederResult, studyType and minVoltageDropProportionalThreshold
+    // VERSION = 1.1 withVoltageMap -> withSimpleResult and withVoltageAndVoltageDropProfileResult
+    public static final String VERSION = "1.1";
+
     private final String id;
 
     private final boolean withLimitViolations;
 
-    private final boolean withVoltageMap;
+    private final boolean withSimpleResult;
```

Comment:
@JB-H and @sylvlecl `withSimpleResult` is maybe not the good naming but it is a way to make the difference between systematic and selective simulations. What I understand is that both type of simulations could provide voltage and voltage drop at buses, but in case of a systematic calculation the voltage is not a `FortescueValue` but the magnitude of the voltage on the direct comment. If you agree with that, we have to document that.

Reply:
Actually, for now, in a systematic study (~withFortescueResults=False), only the magnitude of the current is given. It is just stored in a FortescueValue object.